MaxText/layers/attentions.py

```diff
@@ -198,6 +202,10 @@ def tpu_flash_attention(self, query: Array, key: Array, value: Array, decoder_se
     axis_names = nn.logical_to_mesh_axes(self.flash_axis_names)
     segment_axis_names = nn.logical_to_mesh_axes((BATCH, "activation_length_no_heads"))
 
+    kBLOCK_1 = self.config.sa_block_q
+    kBLOCK_2 = self.config.sa_block_q_dkv
+    kBLOCK_3 = self.config.sa_block_q_dq
+    print(f"KBLOCK VALUES: {kBLOCK_1} {kBLOCK_2} {kBLOCK_3}")
     @functools.partial(
```

Comment:
I don't think this is horrible, I am fine with it. Since wrap_flash_attention is not the top level jit using static_argnums may be even uglier to plumb through